packages/mobile/src/screens/mood-collection-screen/MoodCollectionScreen.tsx

```diff
@@ -45,7 +46,10 @@ export const MoodCollectionScreen = ({
       <ScreenHeader text={`${collection.title} Playlists`} />
       <ScreenContent>
         <WithLoader loading={status === Status.LOADING}>
-          <CollectionList collection={exploreData} />
+          <CollectionList
+            collection={exploreData}
+            style={{ marginBottom: spacing(12) }}
```

Comment:
Does this work when the NowPlaying bar is showing too?

Reply:
yes, good thought though 👍